pkg/services/query/query.go

```diff
@@ -136,6 +138,13 @@ func (s *Service) handleQueryData(ctx context.Context, user *models.SignedInUser
 		Queries: []backend.DataQuery{},
 	}
 
+	middlewares := []httpclient.Middleware{}
+	if parsedReq.httpRequest != nil {
```

Comment:
What is the case when `parsedReq.httpRequest` is nil? seems like things should be HTTP requests here.

Reply:
this branch was pointed at https://github.com/grafana/grafana/pull/50466, so these changes aren't related to this PR